DOL.WHD.Section14c.Web/.eslintrc.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         //Best Practices
         "accessor-pairs": 2,
         "block-scoped-var": 0,
-        "complexity": [2, 6],
+        "complexity": [2, 12],
```

Comment:
Agreed it should be applied to the individual method in question. Just a ham-handed temporary fix that got swept up in a bunch of other changes.